libs/ipc/inc/public/tfc/ipc.hpp

```diff
@@ -103,25 +110,44 @@ class slot {
 
   auto operator=(slot const&) -> slot& = delete;
 
-  [[nodiscard]] auto value() const noexcept -> std::optional<value_t> const& { return slot_->value(); }
+  [[nodiscard]] auto value() const noexcept -> std::optional<value_t> const& { return filters_.value(); }
 
   [[nodiscard]] auto name() const noexcept -> std::string_view { return slot_->name(); }
 
   [[nodiscard]] auto full_name() const noexcept -> std::string { return slot_->name_w_type(); }
```

Comment:
I would look into calling this name_with_type or something more meaningful then full_name.
Maybe unique_name?